views/AnimatedProfile.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react'
+import { View, Text, Image, StyleSheet, TextInput, Dimensions, Animated, Easing, TouchableOpacity } from "react-native"
+import { connect } from "react-redux"
+import Badges from "../components/Badges"
+
+const { height, width } = Dimensions.get("window")
```

Comment:
The "height" and "width" variables are declared globally as they need to be accessed in both the component and the stylesheet (starting at line 88)